modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -4476,6 +4518,31 @@ private void _handle(
 			new ValidationError(objectEntryValuesException.getMessage()));
 	}
 
+	private void _inactiveLatestObjectEntryVersion(ObjectEntry objectEntry)
+		throws PortalException {
```

Comment:
@JhossephG, please add a test to cover the scenario where the penultimate version is set to inactive and the last version is scheduled.